docs/source/sparse.rst

```diff
@@ -363,6 +363,78 @@ assumption that the fill value is negative infinity.
 
 .. See https://github.com/Quansight-Labs/rfcs/tree/pearu/rfc-fill-value/RFC-0004-sparse-fill-value for a new API
 
+.. _sparse-csr-docs:
+
+Sparse CSR Tensor
++++++++++++++++++
+
+The CSR (Compressed Sparse Row) sparse tensor format implements the CSR format
+for storage of 2 dimensional tensors. Although there is no support for N-dimensional
+tensors, the primary advantage over the COO format is better use of storage and
+much faster computation operations such as sparse matrix-vector multiplication
+using MKL and MAGMA backends. CUDA support does not exist as of now.
+
+A CSR sparse tensor consists of three 1-D tensors: ``crow_indices``, ``col_indices``
+and ``values``:
+
+  - The ``crow_indices`` tensor consists of compressed row indices. This is a 1-D tensor
+    of size ``size[0] + 1``. The last element is the number of non-zeros.
```

Comment:
I know people can look it up on Wikipedia, but it would be nice to say what exactly a compressed row indices here is

Reply:
OK done.